soroban-env-host/src/host/conversion.rs

```diff
@@ -269,6 +269,16 @@ impl Host {
         }
         Ok(ScMap(self.map_err(mv.try_into())?))
     }
+
+    pub(crate) fn instance_storage_map_to_scmap(&self, map: &HostMap) -> Result<ScMap, HostError> {
```

Comment:
Nit: Do you mind adding a comment here to explain the difference between it and `host_map_to_scmap` and why it's necessary?

Reply:
Done.